test_runner/fixtures/neon_fixtures.py

```diff
@@ -1998,14 +2001,14 @@ def get_test_output_dir(request: Any) -> str:
 # this fixture ensures that the directory exists.  That works because
 # 'autouse' fixtures are run before other fixtures.
 @pytest.fixture(scope='function', autouse=True)
-def test_output_dir(request: Any) -> str:
+def test_output_dir(request: Any) -> pathlib.Path:
```

Comment:
nit: It seems that both `Path` and `pathlib.Path` are used arbitrarily. Could we stick with just one, perhaps `Path`?